### PR TITLE
feat(schema): add the Python representation of the Portex builtin types

### PR DESCRIPTION
**Pull request by linjiX**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2b4qrac 实现  Portex 内置类型 python 表示](https://app.clickup.com/t/2b4qrac)

File: graviti/schema/builtin.py [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+          fields={
+            'f0': int_(),
+            'f1': float_(
+              minimum=0,
+              maximum=100,
+            ),
+          },
+        )
+
+        Create a record by tuple:
```
Comment (Lee-000) [inline graviti/schema/builtin.py:231]: `by tuple list`?

File: graviti/schema/builtin.py [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+    def __getitem__(self, index: Union[int, str, slice]) -> Union[PortexType, List[PortexType]]:
+        if isinstance(index, slice):
+            return [field.type for field in self.fields[index]]
+
+        if isinstance(index, str):
+            for field in self.fields:
+                if index == field.name:
+                    return field.type
+
+            raise IndexError()
```
Comment (Lee-000) [inline graviti/schema/builtin.py:287]: `KeyError`?

File: graviti/schema/base.py [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+            lines.append(")")
+            return f"\n{indent}".join(lines)
+
+        return f"{lines[0]})"
+
+    def to_pyobj(self) -> Dict[str, Any]:
+        """Dump the instance to a python dict.
+
+        Returns:
+            A Python Dict representation of the Portex type.
```
Comment (Lee-000) [inline graviti/schema/base.py:132]: Why different capitalization of `python dict`?

**Review by graczhual — APPROVED** [review]

**Review by AChenQ — APPROVED** [review]

**Review by Lee-000 — APPROVED** [review]

**Comment by GravitiRob** [comment]
Task linked: [CU-2b4qrac 实现  Portex 内置类型 python 表示](https://app.clickup.com/t/2b4qrac)

**Pull request merged**